tests/themes/run.sh

```diff
@@ -23,6 +23,8 @@ for theme in themes/*/theme.lua; do
 
   sed "s~default/theme~$theme/theme~g" "awesomerc.lua" > "$config_file"
 
+  # Set CMAKE_BINARY_DIR for out-of-tree builds.
+  CMAKE_BINARY_DIR="$PWD" \
```

Comment:
`tests/run.sh` uses it.
It could use something else maybe.
As for `check-integration` there it gets set probably, but when run via `tests/themes/run.sh` (via `check-integration` => `check-themes`) it does not seem to be available?!
So CMake maybe sets it, but not exported?!